srcpkgs/libvterm/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'libvterm'
 pkgname=libvterm
-version=0.0.20151229
-revision=2
-_commit=04c0777b139cfbddb057c0cbfc007677dd9f1b4f
+version=0.1
+revision=1
+_commit=f8ab54cf592b945caaaf2bc397c0da6fc8fb949b
```

Comment:
Original 0.1 https://launchpad.net/libvterm is tagged, can we switch?

Reply:
I'll try it in a moment. Here's [the diff](https://gist.github.com/AluisioASG/87ca7d13fc48593b2b754e93353b2d3f) between upstream 0.1.1 and the nvim branch, for reference.